ui/apps.py

```diff
@@ -21,3 +21,7 @@ def ready(self):
         if missing_settings:
             raise ImproperlyConfigured(
                 'The following settings are missing: {}'.format(', '.join(missing_settings)))
+
+        # write the x509 certification & key to files
+        from ui.utils import write_x509_files
+        write_x509_files()
```

Comment:
I'm asking why we are counting on the app to write these files at all. Why can't we just expect the two files to exist with the correct key/cert? If the keys/certs change, you just change the file in place, and we don't have to deal with the awkwardness of storing those large secret values with `\n` sprinkled throughout in the `.env` file